Documentation/tectonic.md

```diff
@@ -0,0 +1,74 @@
+- A running matchbox node which will do the provisioning (Matchbox is only required if you are running Tectonic on Bare metal)
+- You can run through all steps of the GUI installer, but in the last step, choose `Boot manually`. This way we can make the necessary changes first.
+
+## Edit the kubelet.service file to be compatible with Rook
+
+To make the kubelet compatible with Rook we need to make a few adaptions to the systemd service file generated by the Tectonic-installer.
+
+First change to the directory where you untarred the tectonic installer and find your newly generated cluster configuration files.
+
+```
+ $ cd ~/tectonic/tectonic-installer/LINUX-OR-DARWIN/clusters
+```
+
+
+Open the file `modules/ignition/resources/services/kubelet.service` in your favorite editor and after the last line containing `ExecStartPre=...`, paste the following extra lines:
+
+```
+ExecStartPre=/bin/mkdir -p /var/lib/kubelet/volumeplugins
+ExecStartPre=/bin/mkdir -p /var/lib/rook
+```
+
+And after the `ExecStart=/usr/lib/coreos/kubelet-wrapper \` line, insert the following command line option for the kubelet-wrapper:
+
+```
+--volume-plugin-dir=/var/lib/kubelet/volumeplugins \
+```
+
+Save the file and close.
+
+### Boot your Tectonic cluster
+
+All the preparations are ready for Tectonic to boot now. We will use `terraform` to start the cluster.
+Visit the official [Tectonic manual boot](https://coreos.com/tectonic/docs/latest/install/aws/manual-boot.html#deploy-the-cluster) page for the commands to use.
+
+**Remark:** The Tectonic installer contains the correct terraform binary out of the box. This terraform binary can be found in following directory `~/tectonic/tectonic-installer/linux`.
+
+## Start Rook
+
+After the Tectonic Installer ran and the Kubernetes cluster is started and ready, you can follow the [Rook installation guide](kubernetes.md).
+If you want to specify which disks Rook can use, follow the instructions in [creating Rook clusters](cluster-crd.md)
+
+
+## Enable Prometheus monitoring
+To enable a working Prometheus monitoring cluster in the Tectonic installation, find the files located in `cluster/examples/kubernetes/monitoring/tectonic`
+Before we can enable those files, we need to make a few changes according to your personal setup:
+- `ingress.yaml` change `rook.example.com` into the domain name you want to use
+- `rook-prometheus.yaml` change `rook.example.com` into the domain name you chose in `ingress.yaml`, leaving the `/prometheus` at the end as is
+- `rook-grafana.yaml` change `rook.example.com` into the domain name you chose in `ingress.yaml`, leaving the `/grafana` at the end as is
+- `rook-grafana.yaml` change default login credentials for the Grafana dashboard (default: `admin:admin`). This secret is a base64 encrypted version of the username and the password.
```

Comment:
I don't think there should be Grafana manifests here. That is a task for the user to do.